package.json

```diff
@@ -9,11 +9,12 @@
   "scripts": {
     "check-integrity": "yarn check --integrity && echo 'Ignore paths from lint-staged'",
     "pretest": "sku configure",
-    "test": "yarn check --integrity && yarn lint && yarn build",
+    "test": "yarn check --integrity && yarn lint && yarn build && yarn contract-test",
+    "contract-test": "sku test",
```

Comment:
Given the name of this command i would recommend explicitly passing the path to the contract test here.